Linkbrary/javascript/error.js

```diff
@@ -0,0 +1,49 @@
+// 에러조건
+// id 에러 상황
+function idError(event){ 
+  event.preventDefault(); // 새로고침 막기
+  if(window.location.pathname==='/Linkbrary/signin.html'){
+    if(ID_INPUT.value?.trim()==false){
+      showidErrorMessage("이메일을 입력해 주세요.");
+      return;
+    }
+    else if(!isValidEmail(ID_INPUT.value)){
+      showidErrorMessage("올바른 이메일 주소가 아닙니다.");
+    }
+  }else{
+    if(ID_INPUT.value?.trim()==false){
+      showidErrorMessage("이메일을 입력해 주세요.");
+      return;
+    }
+    else if(!isValidEmail(ID_INPUT.value)){
+      showidErrorMessage("올바른 이메일 주소가 아닙니다.");
+    }
+    else if(ID_INPUT.value === TEST_EMAIL){
+      showidErrorMessage("이미 사용 중인 이메일입니다.");
+    }
+  }
+} 
+// 이메일 유효성 검사
+function isValidEmail(email) { 
+  const emailRegex = new RegExp(/^[^\s@]+@[^\s@]+\.[^\s@]+$/);
+  return emailRegex.test(email);
+}
+// pw 에러 상황
+function pwError(event){ 
+  event.preventDefault(); // 새로고침 막기
+  if(PW_INPUT[0].value?.trim()==false){
```

Comment:
무조건 === 로 표시해주셔요! 그리고 이 경우에는 그냥 ! 로도 표기 가능할 거 같습니다
```suggestion
  if(!PW_INPUT[0].value?.trim()){
```